eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -29,12 +29,10 @@ stages:
                       Get-ChildItem $(Pipeline.Workspace)/staging
                     displayName: Copying ${{artifact.name}} to staging directory
 
-                - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
-
-                - template: pipelines/steps/net-signing.yml@azure-sdk-build-tools
+                - template: pipelines/steps/net-signing.yml
```

Comment:
Does this work? I wouldn't expect it to work this way because it evaluates templates before any of the steps like the checkout above actually happens.

Reply:
Updated